docs/dev/object-deletion.md

```diff
@@ -0,0 +1,118 @@
+    The deletion annotation would be added to this manifest.
+* `0000_50_cluster-svcat-apiserver-operator_02_config.crd.yaml` containing a cluster-scoped CRD.
+    The deletion annotation would be added to this manifest.
+* `0000_50_cluster-svcat-apiserver-operator_03_config.cr.yaml` containing a cluster-scoped, create-only ServiceCatalogAPIServer.
+    The deletion annotation would be added to this manifest.
+* `0000_50_cluster-svcat-apiserver-operator_03_configmap.yaml` containing a ConfigMap in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the ConfigMap would be removed as part of the namespace deletion.
+* `0000_50_cluster-svcat-apiserver-operator_03_version-configmap.yaml` containing another ConfigMap in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the ConfigMap would be removed as part of the namespace deletion.
+* `0000_50_cluster-svcat-apiserver-operator_04_roles.yaml` containing a cluster-scoped ClusterRoleBinding.
+    The deletion annotation would be added to this manifest.
+* `0000_50_cluster-svcat-apiserver-operator_05_serviceaccount.yaml` containing a ServiceAccount in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the ServiceAccount would be removed as part of the namespace deletion.
+* `0000_50_cluster-svcat-apiserver-operator_06_service.yaml` containing a Service in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the Service would be removed as part of the namespace deletion.
+* `0000_50_cluster-svcat-apiserver-operator_07_deployment.yaml` containing a Deployment in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the Deployment would be removed as part of the namespace deletion.
+* `0000_50_cluster-svcat-apiserver-operator_08_cluster-operator.yaml` containing a cluster-scoped ClusterOperator.
+    The deletion annotation would be added to this manifest.
+* `0000_90_cluster-svcat-apiserver-operator_00_prometheusrole.yaml` containing a Role in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the Role would be removed as part of the namespace deletion.
+* `0000_90_cluster-svcat-apiserver-operator_01_prometheusrolebinding.yaml` containing a RoleBinding in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the RoleBinding would be removed as part of the namespace deletion.
+* `0000_90_cluster-svcat-apiserver-operator_02-operator-servicemonitor.yaml` containing a ServiceMonitor in the `openshift-service-catalog-apiserver-operator` namespace.
+    This manifest would be dropped, because the ServiceMonitor would be removed as part of the namespace deletion.
+
```

Comment:
We need to restructure the above as it is hard to read as there are so many manifest names. We can create two broad categories
* The deletion annotation should be added to these manifests below
* These manifests should be dropped

Reply:
done